engine/src/main/java/org/hibernate/search/annotations/IndexedEmbedded.java

```diff
@@ -34,9 +34,9 @@
 	String DEFAULT_NULL_TOKEN = "__DEFAULT_NULL_TOKEN__";
 
 	/**
-	 * Field name prefix, defaults to {@code propertyname.}.
+	 * Field name prefix.
 	 *
-	 * @return the field name prefix. Default to "."
+	 * @return the field name prefix. Default to ".", indicating that {@code propertyname.} should be used.
```

Comment:
No I think the existing wording is OK. The context where "should" is used is not the same.

But see "the token {@code _null_} will be used" in what you quoted.